sdk/location/src/main/java/com/klaviyo/location/KlaviyoGeofence.kt

```diff
@@ -114,7 +124,12 @@ fun JSONObject.toKlaviyoGeofence(): KlaviyoGeofence? = try {
         id = getString(KlaviyoGeofence.KEY_ID),
         latitude = getDouble(KlaviyoGeofence.KEY_LATITUDE),
         longitude = getDouble(KlaviyoGeofence.KEY_LONGITUDE),
-        radius = getDouble(KlaviyoGeofence.KEY_RADIUS).toFloat()
+        radius = getDouble(KlaviyoGeofence.KEY_RADIUS).toFloat(),
+        duration = if (has(KlaviyoGeofence.KEY_DURATION) && !isNull(KlaviyoGeofence.KEY_DURATION)) {
```

Comment:
and if we do `duration = attributes.optInt("duration").takeIf { it > 0 }` above, may as well do it here too.